src/riscv/etherlink/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[workspace]
+members = ["kernel"]
+resolver = "2"
+
+[patch.crates-io]
+tezos-smart-rollup = { git = "https://gitlab.com/tezos/tezos.git", branch = "vdum@rv-625" }
+tezos-smart-rollup-host = { git = "https://gitlab.com/tezos/tezos.git", branch = "vdum@rv-625" }
+tezos-smart-rollup-core = { git = "https://gitlab.com/tezos/tezos.git", branch = "vdum@rv-625" }
+tezos-smart-rollup-mock = { git = "https://gitlab.com/tezos/tezos.git", branch = "vdum@rv-625" }
+tezos-smart-rollup-encoding = { git = "https://gitlab.com/tezos/tezos.git", branch = "vdum@rv-625" }
+tezos-smart-rollup-entrypoint = { git = "https://gitlab.com/tezos/tezos.git", branch = "vdum@rv-625" }
+tezos-smart-rollup-debug = { git = "https://gitlab.com/tezos/tezos.git", branch = "vdum@rv-625" }
+
+[workspace.dependencies]
+evm_kernel = { git = "https://gitlab.com/tezos/tezos.git", branch = "vdum@rv-625" }
```

Comment:
I think it'll be easier if the Etherlink changes merge on GitLab. With that in place, you only need to point the dependnecy at the default GitLab URL and you don't need patches.

Reply:
Yes, I'll update this once the corresponding MR gets merged.